Patterns/English/English-NumbersWithUnit.yaml

```diff
@@ -502,8 +502,10 @@ AmbiguousDimensionUnitList: !list
 #EnglishNumberWithUnitExtractorConfiguration
 BuildPrefix: !simpleRegex
   def: (?<=(\s|^|\W))
+  def_js: (?=(\s|^|\W)|(?!(\s|^|\W))\b)
 BuildSuffix: !simpleRegex
   def: (?=(\s|\W|$))
+  def_js: (?=(\s|\W|$))
```

Comment:
For the cases where the js regex is the same as the regular one, can't it just reference the value of the other, instead of duplicating the regex text?